docs/ML_DGA.md

```diff
@@ -0,0 +1,61 @@
+# Machine Learning on Domain Generation Algorithm (DGA)
+
+Several blogs were put out on how you can create and leverage supervised DGA ML models to enrich data within the stack.
+* Part 1: [Machine learning in cybersecurity: Training supervised models to detect DGA activity](https://www.elastic.co/blog/machine-learning-in-cybersecurity-training-supervised-models-to-detect-dga-activity)
+* Part 2: [Machine learning in cybersecurity: Detecting DGA activity in network data](https://www.elastic.co/blog/machine-learning-in-cybersecurity-detecting-dga-activity-in-network-data)
+
+You can also find some supplementary and examples [here](https://github.com/elastic/examples/tree/master/Machine%20Learning/DGA%20Detection)
+
+For questions please reach out to the ML team in the #machine-learning channel of the 
```

Comment:
I would also add the Stack ML tag in the discuss forums as a potential way to reach us
https://discuss.elastic.co/tags/c/elastic-stack/81/stack-machine-learning

Reply:
Good call, thanks!